test/testcafe/framework/page-objects/ChallengeOktaVerifyPushPageObject.js

```diff
@@ -1,9 +1,9 @@
 import ChallengeFactorPageObject from './ChallengeFactorPageObject';
 
 const FORM_INFOBOX_WARNING = '.okta-form-infobox-warning';
-const FORM_INFOBOX_ERROR = '.o-form-error-container .infobox-error';
+const FORM_INFOBOX_ERROR = '[data-se="o-form-error-container"] [data-se="callout"]';
```

Comment:
like you had said the data-se in incomplete. tHis works for our usecase but .infobox-error is more accurate.

Reply:
hmm not sure what you are suggesting here ?

Reply:
what I meant is [data-se="callout"] is not as specific as ..infobox-error but I guess it is okay in this case.